README.md

```diff
@@ -161,6 +161,7 @@ _For a complete listing of all available packages, see [Django Packages](https:/
 - [django-cleanup](https://github.com/un1t/django-cleanup) - Zero configuration file/image removal for local and remote files.
 
 ### Forms
+- [django-bleach](https://github.com/marksweb/django-bleach/) - Integrate [bleach](https://github.com/mozilla/bleach) into forms and models.
```

Comment:
We'd either need to merge the duplicate packages or drop the link here. 

```suggestion
- [django-bleach](https://github.com/marksweb/django-bleach/) - Integrate bleach into forms and models.
```

Reply:
I didn't want to remove the original bleach link, it's worth keeping as there may be other ways to implement besides form and model fields. But if someone was reading through and didn't know what bleach is, the link here would help explain.

Reply:
@jefftriplett Ah I've just seen the linter fail on a double link so I've removed the 2nd link.